resources/views/page-builder/blog_overview.blade.php

```diff
@@ -1,5 +1,5 @@
 @php($blogs = \Statamic\Statamic::tag('collection:blog')->fetch())
-<div class="relative py-16">
+<div class="relative py-16 component blog-overview">
```

Comment:
should the component class be positioned first, since everything else is tailwind classes?